browser-sign-in/OktaBrowserSignIn/AppDelegate.swift

```diff
@@ -15,28 +15,89 @@
  */
 
 import UIKit
-import OktaOidc
+import WebAuthenticationUI
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-    
     var window: UIWindow?
-
+    var configForUITests: [String: String]? {
+        let env = ProcessInfo.processInfo.environment
+        guard let oktaURL = env["OKTA_URL"], oktaURL.count > 0,
+              let clientID = env["CLIENT_ID"],
+              let redirectURI = env["REDIRECT_URI"],
+              let logoutRedirectURI = env["LOGOUT_REDIRECT_URI"]
```

Comment:
You might need to make sure these are not `isEmpty`, since the environment variables may just return empty strings.